_posts/2020-07-07-delving-into-docker.md

```diff
@@ -1,3 +1,10 @@
+---
+ title: "Delving into Docker"
+ tags: docker containers
+ author: Bhargav SNV Aditi Ahuja
```

Comment:
Also change this line to:
Bhargav SNV and Aditi Ahuja
Not sure if that^ will work, but this definitely won't.